pom.xml

```diff
@@ -104,6 +104,17 @@
             <version>0.11.5</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <version>0.8.10</version>
```

Comment:
Please move version to properties and use it as variable . It should be done as mapstruct ```mapstruct.version```